requirements.txt

```diff
@@ -7,56 +7,53 @@
 amqp==2.1.4
 anymarkup-core==0.7.1     # via anymarkup
 anymarkup==0.7.0
-argcomplete==1.8.2        # via selinonlib
 beautifulsoup4==4.6.0
-bigquery-python==1.12.0
-billiard==3.5.0.2         # via celery
-boto3==1.4.8
+bigquery-python==1.13.0
+billiard==3.5.0.3         # via celery
+boto3==1.5.28
 boto==2.48.0
-botocore==1.8.16          # via boto3, s3transfer
-celery==4.0.2             # via selinon, selinonlib
-certifi==2017.4.17        # via requests
+botocore==1.8.42          # via boto3, s3transfer
+celery==4.1.0             # via selinon
 chardet==3.0.4            # via requests
-codegen==1.0              # via selinonlib
+click==6.7                # via selinon
+codegen==1.0              # via selinon
 colorama==0.3.9           # via rainbow-logging-handler
 configobj==5.0.6          # via anymarkup
-docutils==0.13.1          # via botocore
-flexmock==0.10.2          # via selinonlib
+docutils==0.14            # via botocore
 git2json==0.2.3
-google-api-python-client==1.6.2  # via bigquery-python
-graphviz==0.8             # via selinonlib
+google-api-python-client==1.6.5  # via bigquery-python
+graphviz==0.8.2           # via selinon
 httplib2==0.10.3          # via bigquery-python, google-api-python-client, oauth2client
-idna==2.5                 # via requests
+idna==2.6                 # via requests
 jmespath==0.9.3           # via boto3, botocore
 jsl==0.2.4
-json5==0.2.4              # via anymarkup
+json5==0.6.0              # via anymarkup
 jsonschema==2.6.0
-kombu==4.0.2              # via celery
+kombu==4.1.0              # via celery
```

Comment:
This might be a problem because we install our patched 4.0.2 version in [worker-base](https://github.com/fabric8-analytics/fabric8-analytics-worker-base/blob/master/Dockerfile#L28) so this results in

```
Found existing installation: kombu 4.0.2
    Uninstalling kombu-4.0.2:
      Successfully uninstalled kombu-4.0.2
Successfully installed kombu-4.1.0
```

while previously it was

```
Requirement already satisfied: kombu==4.0.2 in /usr/lib/python3.4/site-packages
```

So either @msrb needs to rebase his https://github.com/msrb/kombu to latest upstream or we need to keep kombu pinned to 4.0.2 here.

Reply:
I can try to pin it here to see whether it will resolve/work.

Reply:
@pkajaba Let's try the latest version. The upstream issue is already closed and the fix should be included in the latest release. We don't want to maintain our kombu fork (https://github.com/fabric8-analytics/fabric8-analytics-worker/issues/106)

Reply:
Thanks Michal. [worker-base PR for removing the forked kombu](https://github.com/fabric8-analytics/fabric8-analytics-worker-base/pull/29)